addons/interact_menu/initSettings.sqf

```diff
@@ -7,3 +7,57 @@
     false,
     {GVAR(selectorColorHex) = _this call BIS_fnc_colorRGBtoHTML} // Stored in Hex to avoid constant conversion
 ] call CBA_settings_fnc_init;
+
+[
+    QGVAR(alwaysUseCursorInteraction),
+    "CHECKBOX",
+    localize LSTRING(AlwaysUseCursorInteraction),
+    [format ["ACE %1", localize LSTRING(Category_InteractionMenu)], localize LSTRING(Category_InteractionMenu)],
+    false,
+    false
+] call CBA_settings_fnc_init;
+
+[
+    QGVAR(cursorKeepCentered),
+    "CHECKBOX",
+    [LSTRING(cursorKeepCentered), LSTRING(cursorKeepCenteredDescription)],
+    [format ["ACE %1", localize LSTRING(Category_InteractionMenu)], localize LSTRING(Category_InteractionMenu)],
+    false,
+    false
+] call CBA_settings_fnc_init;
+
+[
+    QGVAR(menuBackground),
+    "LIST",
+    localize LSTRING(background),
+    [format ["ACE %1", localize LSTRING(Category_InteractionMenu)], localize LSTRING(Category_InteractionMenu)],
+    [[0, 1, 2], ["STR_A3_OPTIONS_DISABLED", localize LSTRING(backgroundBlur), localize LSTRING(backgroundBlack)], 0],
+    false
+] call CBA_settings_fnc_init;
+
+[
+    QGVAR(alwaysUseCursorSelfInteraction),
+    "CHECKBOX",
+    localize LSTRING(AlwaysUseCursorInteraction),
+    [format ["ACE %1", localize LSTRING(Category_InteractionMenu)], localize ELSTRING(Interaction,InteractionMenuSelf)],
+    true,
+    false
+] call CBA_settings_fnc_init;
+
+[
+    QGVAR(cursorKeepCenteredSelfInteraction),
+    "CHECKBOX",
+    [LSTRING(cursorKeepCentered), LSTRING(cursorKeepCenteredDescription)],
+    [format ["ACE %1", localize LSTRING(Category_InteractionMenu)], localize ELSTRING(Interaction,InteractionMenuSelf)],
+    true,
+    false
+] call CBA_settings_fnc_init;
+
+[
+    QGVAR(menuBackgroundSelf),
+    "LIST",
+    localize LSTRING(background),
```

Comment:
afaik CBA settings also calls localize internally if it finds a localize able string.

Reply:
Looks like it does, I'll fix that up